expression/constant_propagation_test.go

```diff
@@ -15,6 +15,7 @@ package expression_test
 
 import (
 	"fmt"
+	"github.com/pingcap/tidb/util/testutil"
```

Comment:
nit: the order of import packages should follow the lexicographical order, reference: https://github.com/golang/go/wiki/CodeReviewComments#imports